github-download-release/action.yml

```diff
@@ -0,0 +1,62 @@
+name: Github Download Release
+description: Downloads the specified release using gh client
+inputs:
+  token:
+    description: The authorization token for the target repo
+    required: true
+  github-version:
+    description: Github version
```

Comment:
if this is for the GH CLI it might be good to specify that; `gh-version` or something

Reply:
Correct this is for the gh cli client. I like your suggestion.